lightning/config/config.go

```diff
@@ -479,6 +479,23 @@ func (cfg *Config) Adjust() error {
 		}
 	}
 
+	// adjust file routing
+	for _, rule := range cfg.Mydumper.FileRouters {
+		if rule.Path != "" {
+			if filepath.IsAbs(rule.Path) {
```

Comment:
the `!= ""` check is redundant since `filepath.IsAbs("")` is false on both Linux and Windows.

```suggestion
		// if rule.Path is empty, filepath.IsAbs("") is false and won't enter this branch for absolute-to-relative adjustment.
		if filepath.IsAbs(rule.Path) {
```